core/templates/dev/head/library/library.html

```diff
@@ -45,6 +45,12 @@
         </div>
 
         <div class="oppia-exp-summary-tiles-container" ng-if="!inSearchMode">
+          <img src="/images/library/banner.svg" align="center" width="100%">
+          <div style="text-align: center; width: 100%;">
```

Comment:
Same here, might want to consider moving these style settings into the CSS file.